app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,72 @@
+<div class="d-block d-md-flex listing-horizontal">
+  <li id="post-<%= post.id %>">
+
+    <%= link_to gravatar_for(post.user, size: Settings.user.gravatar_post_size), post.user %>
+    <span class="user"><%= link_to post.user.username, post.user %></span>
+
+    <span class="timestamp">
+      <%= t("label.posted") %> <%= t time_ago_in_words(post.created_at)%> <%= t("label.ago") %>.
```

Comment:
` <%= t time_ago_in_words(post.created_at)%> <%= t("label.ago") %>.`
Chú tìm hiểu về `format datetime ` với I18n nào https://guides.rubyonrails.org/i18n.html#how-to-store-your-custom-translations